apps/doji-frontend/src/pages/session-v2/[id]/index.tsx

```diff
@@ -0,0 +1,178 @@
+import { httpClient } from '@frontend/services'
+import { ExtendedNextPage } from '@frontend/type'
+import { IPublicSessionReviewDTO, IReviewStat, ISessionInformationDTO } from '@libs/api'
+import {
+  Avatar,
+  Button,
+  CircularProgress,
+  Grid,
+  LinearProgress,
+  Link,
+  Stack,
+  TextField,
+  Typography,
+  useTheme,
+} from '@mui/material'
+import { Box } from '@mui/system'
+import { formatDistanceToNow } from 'date-fns'
+import { useRouter } from 'next/router'
+
+import React from 'react'
+import { AiFillStar, AiOutlineStar } from 'react-icons/ai'
+import { BsThreeDotsVertical } from 'react-icons/bs'
+import { useQuery } from 'react-query'
+
+function SessionDetail() {
+  return (
+    <Box>
+      <Stack direction="row" py="0.5em" justifyContent="space-between">
+        <Stack spacing="0.5em">
+          <Typography variant="large" fontWeight={700}>
+            From 1M to 1K, How did I do it?
+          </Typography>
+          <Stack direction="row" spacing="0.5em">
+            <Avatar src="/static/images/avatar/1.jpg" sx={{ width: 24, height: 24 }} />
```

Comment:
avoid using image in this repo. Try to use online image instead such as https://mui.com/static/images/avatar/1.jpg

Reply:
TIL. Why is that?

Reply:
![image](https://user-images.githubusercontent.com/16436051/160230630-5c99f4ba-6179-4528-85dc-a160277352ac.png)
 Now it looks boring. :cry:

Reply:
https://mui.com/static/images/avatar/2.jpg
and
https://mui.com/static/images/avatar/3.jpg

Reply:
https://mui.com/static/images/avatar/4.jpg

Reply:
Ok!. added more variety.